cypress/integration/eventplanner/editall.spec.js

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
+
+context('Editing', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  it('edits a packlist', () => {
```

Comment:
This test is very long. I think you could split it into multiple separate tests. A lot of the steps seem unrelated to editing the list, you just check if some elements exist.

Reply:
I tried to make it easier, but it is difficult, to get the right element to use it.